okio/src/main/java/okio/BufferedSource.java

```diff
@@ -499,4 +500,10 @@ boolean rangeEquals(long offset, ByteString bytes, int bytesOffset, int byteCoun
 
   /** Returns an input stream that reads from this source. */
   InputStream inputStream();
+
+  /** Returns a Reader that reads from this source. */
```

Comment:
Don't like as in:
a) let's get rid of it
b) rename it to readerUtf8()
c) something else entirely?